tensorflow_model_optimization/python/core/sparsity_tf2/pruning_optimizer.py

```diff
@@ -12,31 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-"""Keras callbacks for pruning."""
+"""Meta-optimizer to interfece with custom sparse training pruners."""
```

Comment:
"Optimizer wrapper that adds sparsity training via pruning to other optimizers."